pkg/types/openstack/validation/platform.go

```diff
@@ -31,24 +32,23 @@ func ValidatePlatform(p *openstack.Platform, n *types.Networking, fldPath *field
 		} else if !isValidValue(p.FlavorName, validFlavors) {
 			allErrs = append(allErrs, field.NotSupported(fldPath.Child("computeFlavor"), p.FlavorName, validFlavors))
 		}
+		p.TrunkSupport = "0"
 		netExts, err := fetcher.GetNetworkExtensionsAliases(p.Cloud)
 		if err != nil {
-			allErrs = append(allErrs, field.InternalError(fldPath.Child("trunkSupport"), errors.New("could not retrieve networking extension aliases")))
+			logrus.Warning("Could not retrieve networking extension aliases. Assuming trunk ports are not supported.")
 		} else {
 			if isValidValue("trunk", netExts) {
 				p.TrunkSupport = "1"
-			} else {
-				p.TrunkSupport = "0"
 			}
 		}
+		p.OctaviaSupport = "0"
 		serviceCatalog, err := fetcher.GetServiceCatalog(p.Cloud)
 		if err != nil {
-			allErrs = append(allErrs, field.InternalError(fldPath.Child("octaviaSupport"), errors.New("could not retrieve service catalog")))
+			logrus.Warning("Could not retrieve service catalog. Assuming there is no Octavia load balancer service available.")
+			p.OctaviaSupport = "0"
```

Comment:
If the default value of no support is preserved, why are you setting again?

Reply:
oh, right.